src/views/catalog/CustomizeInstanceType/components/CustomizeITVMFooter.tsx

```diff
@@ -28,6 +28,7 @@ import './CustomizeITVMFooter.scss';
 const CustomizeITVMFooter: FC = () => {
   const { t } = useKubevirtTranslation();
   const navigate = useNavigate();
+  const [activeNamespace] = useActiveNamespace();
```

Comment:
Can this page be rendered on 'all-namespaces' ? In this case activeNamespace will be `ALL_NAMESPACES_SESSION_KEY`

Reply:
should be fine, as this is only for the cancel button, but a wider check and adjustment will be followed so changes will be made again